metadoc-js/src/main/scala/metadoc/MetadocFetch.scala

```diff
@@ -5,20 +5,22 @@ import scala.concurrent.Future
 import scala.meta.internal.{semanticdb => s}
 import metadoc.MetadocApp._
 import metadoc.{schema => d}
+import MetadocEnrichments._
 
 object MetadocFetch {
 
   def symbol(symbolId: String): Future[Option[d.SymbolIndex]] = {
-    val url = "symbol/" + JSSha512.sha512(symbolId)
```

Comment:
Does this mean we can remove the js-sha512 NPM dependency and the Scala.js binding?